features/step_definitions/user_sign_up_steps.rb

```diff
@@ -0,0 +1,20 @@
+Given(/^I am on the root page$/) do
+  visit root_path
+end
+
+Given(/^I click link "([^"]*)"$/) do |link|
+  click_link link
+end
+
+Given(/^I am on the registration page$/) do
+  visit new_user_registration_path
+end
+
+Then(/^I select "([^"]*)" from "([^"]*)"$/) do |option, role|
```

Comment:
make these more generic so your colleagues will understand what is going on here - something like `option, choices`